data/docs.yml

```diff
@@ -163,6 +163,15 @@
       user: apache
       repo: skywalking-kubernetes-event-exporter
 
+    - name: SkyWalking Rover
+      icon: skywalking-rover
```

Comment:
There is no rover logo, I think.

Reply:
@Jtrust Could we build one?

Reply:
Ohh, Deleted.

Reply:
How about 'skywalking' instead of 'skywalk-Rover' for the time being?

Reply:
Yes, we could.

@mrproliu You should not remove it, we should use the skywalking as default